src/main/java/data/TaskManager.java

```diff
@@ -369,16 +439,16 @@ public void markManager(WeekView weekView, MonthView monthView, boolean inMonthV
 
     /**
      * Finds the date based on the day number.
-     * 
+     *
      * @param weekView WeekView object for finding the date.
      * @param monthView MonthView object for finding the date.
      * @param inMonthView A boolean indicating whether the view is in month view or not.
      * @param dayInt The day number to find the date for.
      * @return The date corresponding to the day number.
      * @throws TaskManagerException If the date is not in the current month or week being viewed.
      */
-    private static LocalDate findDateFromDayNumber(WeekView weekView, MonthView monthView, 
-            boolean inMonthView, int dayInt) throws TaskManagerException {
+    private static LocalDate findDateFromDayNumber(WeekView weekView, MonthView monthView,
+                                                   boolean inMonthView, int dayInt) throws TaskManagerException {
```

Comment:
I know the checkstyle does not show an error as it only expects either, 8, 46 or 50 space indentation. But to maintain consistency lets stick with the same style!